includes/scheduler/class-post.php

```diff
@@ -51,28 +51,26 @@ function ( $post_id ) {
 	public static function schedule_post_activity( $new_status, $old_status, $post ) {
 		$post = get_post( $post );
 
-		if ( ! $post || is_post_disabled( $post ) ) {
+		if ( ! $post ) {
 			return;
 		}
 
 		if ( 'ap_extrafield' === $post->post_type ) {
-			self::schedule_profile_update( $post->post_author );
+			Actor::schedule_profile_update( $post->post_author );
 			return;
 		}
 
 		if ( 'ap_extrafield_blog' === $post->post_type ) {
-			self::schedule_profile_update( 0 );
+			Actor::schedule_profile_update( 0 );
 			return;
 		}
 
-		// Do not send activities if post is password protected.
-		if ( \post_password_required( $post ) ) {
+		if ( is_post_disabled( $post ) ) {
 			return;
 		}
 
-		// Check if post-type supports ActivityPub.
-		$post_types = \get_post_types_by_support( 'activitypub' );
-		if ( ! \in_array( $post->post_type, $post_types, true ) ) {
```

Comment:
Checking for post type support is now part of `is_post_disabled()`

Reply:
I can't find this check in the `is_post_disabled` function!?!